README.md

```diff
@@ -112,20 +112,6 @@ then tag it with
     <tag name="massive_build.builder" />
 </service>
 ````
-## Accessing the container
-
-You can access the service container by implementing the
-`ContainerAwareInterface`:
-
-````php
-<?php
-// ...
-
-class MyBuilder implements ContainerAwareInterface, BuilderInterface
-{
-    // ...
-}
-````
```

Comment:
Removing this also means that we can stop supporting our own `ContainerAwareInterface`

Reply:
There is still support via `Massive\Bundle\BuildBundle\ContainerAwareInterface` for this but I think we can still remove it from the README as the way to go should be dependency injection.

Reply:
Yes, in a later version we could just stop supporting this feature altogether. Not part of this PR.